### PR TITLE
Created release notes for release 7.5.1

### PR DESCRIPTION
**Pull request by andsel**
Created release notes for release 7.5.1 using the script `tools/release/generate_release_notes.rb` but without the part to autocommit and push

### CODE REVIEW

File: docs/static/releasenotes.asciidoc [hunk 1]
```diff
@@ -22,6 +23,87 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-5-1]]
+=== Logstash 7.5.1 Release Notes
+
+=== Logstash Pull Requests with label v7.5.1
```
Comment (robbavey) [inline docs/static/releasenotes.asciidoc]: This line can be removed

**Comment by karenzone** [comment]
@elasticmachine, run elasticsearch-ci/docs

File: docs/static/releasenotes.asciidoc [hunk 2]
```diff
@@ -22,6 +23,83 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+
```
Comment (robbavey) [inline docs/static/releasenotes.asciidoc]: You are missing the Header here - 

```suggestion
[[logstash-7-5-1]]
=== Logstash 7.5.1 Release Notes
```

**Comment by karenzone** [comment]
@andsel Thank you so much for generating the release notes!  I will make some fixes that will get our docs-ci to green.

**Comment by karenzone** [comment]
@andsel Thank you so much for generating the release notes.  I will make some fixes that will get our docs-ci to green.

**Comment by karenzone** [comment]
@andsel I took your work in this PR and added to it in #11448

**Comment by andsel** [comment]
superseded by #11448

**Pull request closed without merging**